packages/docs/concepts/menus.yaml

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020-2022 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/general.yaml.njk
+  vars:
+    pageId: menus
+    pageTitle: Menus
+    section: Concepts
+    filePath: concepts/menus.yaml
+    content:
+      - id: menu_filter_alert
+        type: Alert
+        properties:
+          type: info
+          showIcon: false
+          message: The menu items are filtered according to pages the user is authorized to see. This can be controlled by making pages public or private, or by assigning roles to authenticated users. See <a href = "./protected-pages">protected pages</a> and <a href = "./roles">roles</a> for more on how to set this up.
+
+      - id: md1
+        type: MarkdownWithCode
+        properties:
+          content: |
+            Menu objects are objects with links to pages. They are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+
+            Blocks like `PageSiderMenu` use the menu with `id: default` by default. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured.
```

Comment:
By default blocks using a menu, like `PageSiderMenu`, will use the menu with `id: default`. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured and referenced under the block properties, allowing for page specific menu configurations.

Reply:
Change made in 0f8e3ebcba1315a5f14cec4a02cd6f8426f948e2